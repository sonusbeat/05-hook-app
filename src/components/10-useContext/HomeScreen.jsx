const HomeScreen = () => {
  return (
    <h1 className="text-center">Home Screen</h1>
  );
}

export default HomeScreen;