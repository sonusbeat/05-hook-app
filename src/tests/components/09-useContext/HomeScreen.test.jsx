import { mount } from 'enzyme';

import HomeScreen from '../../../components/10-useContext/HomeScreen';
import UserContext from '../../../components/10-useContext/UserContext';

describe('Pruebas en <HomeScreen />', () => {
  const user = {
    name: "Daniel",
    email: "user@domain.com"
  };

  const wrapper = mount(
    <UserContext.Provider value={{ user }}>
      <HomeScreen/>
    </UserContext.Provider>
  );

  test('Deberia coincidir con el Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
