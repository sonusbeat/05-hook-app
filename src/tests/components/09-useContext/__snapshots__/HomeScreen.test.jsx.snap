// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen /> Deberia coincidir con el Snapshot 1`] = `
<main>
  <h1
    className="text-center"
  >
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "Daniel",
   "email": "user@domain.com"
}
  </pre>
</main>
`;
